example_test.go

```diff
@@ -27,3 +29,63 @@ func ExampleNewDecoder() {
 		log.Println(card.PreferredValue(vcard.FieldFormattedName))
 	}
 }
+
+// encoding a vcard can be done using the following method
+
+func ExampleNewEncoder() {
+	destFile, err := os.Create("cards.vcf")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer destFile.Close()
+
+	// data in order: first name, middle name, last name, telephone number
+	contacts := [][4]string{
+		{"John", "Webber", "Maxwell", "(+1) 199 8714"},
+		{"Donald", "", "Ron", "(+44) 421 8913"},
+		{"Eric", "E.", "Peter", "(+37) 221 9903"},
+		{"Nelson", "D.", "Patrick", "(+1) 122 8810"},
+	}
+
+	var (
+		// card is a map of strings to []*vcard.Field objects
+		card vcard.Card
+
+		// buffer the writes to save disk I/O time
+		writeBuffer = bufio.NewWriter(destFile)
```

Comment:
Since this is an example, you can elude this. Just write directly to the file, that reduces the noise.